### PR TITLE
Utils

### PR DESCRIPTION
**Pull request by beckynevin**
This update moves a lot of the imports out of Citizen_Science_Testing.ipynb into utils.py. This is different from the functions in the SDK notebook, which is much more behind the scenes. Instead, utils.py has the plotting tools and such.

### CODE REVIEW

**Comment by bnord** [comment]
@beckynevin Could you briefly walk through (during group meeting) the changes to be made. It's hard to parse the jupyter notebook markdown in the web format.

Is this aimed to be immediately moved to script files?

**Comment by jsv1206** [comment]
Could you add these changes i.e. calling utils.py to the latest Citizen_Science_Testing.ipynb and create a pull request?
